### PR TITLE
Add some utility bash scripts

### PR DESCRIPTION
**Pull request by joelstobart-moj**

### CODE REVIEW

File: run-local.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#
+# This script is used to run the Create and Vary a licence API locally, to interact with
```
Comment (timharrison-moj) [inline run-local.sh:2]: title text might need a name change?

**Review by timharrison-moj — APPROVED** [review]
LGTM- just one comment

File: run-local.sh (hunk 1 above)
Comment (joelstobart-moj) [reply run-local.sh:2]: Good point well made :)

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**